assets/tests/state.test.ts

```diff
@@ -447,10 +447,16 @@ describe("reducer", () => {
       State.createRouteTab()
     )
 
+    const expectedNewTab = routeTabFactory.build({
+      isCurrentTab: true,
+      ordering: 5,
+    })
+    delete expectedNewTab.uuid
```

Comment:
**suggestion (non-blocking):** could this be `expectedNewTabFields = {isCurrentTab:true, ordering:5}`, so you can skip building the rest of the object?

Reply:
So I considered that, but I want to make sure that we do match on any other default fields that a new route tab might have, and those will be added to the `routeTab` factory if we ever add more.